modules/src/clients/ics07_tendermint/client_def.rs

```diff
@@ -107,7 +107,7 @@ impl ClientDef for TendermintClient {
             untrusted_state,
             trusted_state,
             &options,
-            Time(chrono::Utc::now()),
+            OffsetDateTime::now_utc().try_into().unwrap(),
```

Comment:
Should we not perhaps implement a `now_utc()` method for the `Time` struct to avoid directly importing dependencies from the `time` crate? I'd generally prefer to use interfaces that hide their underlying implementations as far as possible, as it reduces entanglement.

Reply:
The trouble with `now_utc` is, it requires `std`. So I assume it can't be part of Tendermint's core API without an `std` feature gate, too.

Reply:
Yeah I can look into providing that mechanism. It's probably preferable to avoid mixing newtypes and the types they're supposed to wrap.

Reply:
We'd still want to provide the `TryFrom`/`Into` conversions from/to `OffsetDateTime` for convenience, and `time` is not an optional dependency at the moment (though it could be after redoing the internal representation of `Time` per [ADR-010](https://github.com/informalsystems/tendermint-rs/blob/master/docs/architecture/adr-010-time-api-sanity.md)).